internal/pkg/cli/task_run.go

```diff
@@ -134,6 +176,155 @@ func (o *runTaskOpts) Ask() error {
 	return nil
 }
 
+// Execute create or update task resources and run the task.
+func (o *runTaskOpts) Execute() error {
+	if o.image == "" && o.dockerfilePath == "" {
+		o.dockerfilePath = defaultDockerfilePath
+	}
+
+	if o.imageTag == "" {
+		o.imageTag = defaultImageTag
+	}
+
+	if err := o.getNetworkConfig(); err != nil {
+		return err
+	}
+
+	cluster, err := o.getCluster()
+	if err != nil {
+		return err
+	}
```

Comment:
I used to place it right before `o.runTask`. But thenI thought since `cluster` is a `copilot env`-related config, if there are some issues with that, we should let the user know earlier than later. Any thoughts?

Reply:
make sense 👍